docs/src/setup/configuration.md

```diff
@@ -44,6 +44,22 @@ Please refer to the [Canboat PGN database](https://canboat.github.io/canboat/can
 
 **_NMEA0183_**: The processing of NMEA0183 sentences is done by [nmea0183-signalk](https://github.com/SignalK/signalk-parser-nmea0183)
 
+#### NMEA 0183 Options
+
+- _Suppress nmea0183out event_ - All incoming NMEA0183 data is made available over TCP on port 10110 by default. This happens by incoming data emitting _nmea0183out_ event. Suppressing the event will prevent data from this connection to being routed to TCP.
```

Comment:
All incoming NMEA0183 data, by default, is made available over TCP on port 10110. The  _nmea0183out_ event is emitted when data is output over TCP. Selecting this option will prevent data from this connection appearing on TCP the TCP connection.

Reply:
So this should instead read...
All incoming NMEA0183 data, by default, is made available over TCP on port 10110. Selecting this option will prevent data from this connection from appearing on the TCP connection.

Reply:
I don't know where I got the nmea0183out, because the UI actually says _Suppress nmea0183 event_. But anyway I updated the text.